SalesforceSDK/Salesforce.SDK.TypeScript/data/bootconfig.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
Ideally, it would be nice if we could follow the submodule approach like we do on ```Android``` and ```iOS```, for the shared assets, such as this. Is it possible to check out the submodule of the ```shared``` repo into ```external/shared``` like the other two platforms, and then copy them over/use symlinks using a script?